src/main/resources/application.yml

```
@@ -35,4 +31,7 @@ server:
 #  endpoints:
 #    web:
 #      exposure:
-#        include: refresh
\ No newline at end of file
+#        include: refresh
+
+slack:
+  bot-token: ${SLACK_BOT_TOKEN}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)